core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/version/CodeSystemVersionCreateRequest.java

```diff
@@ -92,11 +90,11 @@
 	// local execution variables
 	private transient Multimap<DatastoreLockContext, DatastoreLockTarget> lockTargetsByContext;
 	private transient Map<String, CodeSystem> codeSystemsByShortName;
+	private transient Date effectiveTimeDate;
 	
 	@Override
 	public Boolean execute(ServiceProvider context) {
-		final RemoteJob job = context.service(RemoteJob.class);
-		final String user = job.getUser();
+		final String user = context.service(User.class).getUsername();
```

Comment:
Do we have the `User` injected for requests that are running in a remote job?

Reply:
Yes, otherwise this would fail.